app/lint-baseline.xml

```diff
@@ -56,7 +56,7 @@
         id="InvalidPackage"
         message="Invalid package reference in library; not included in Android: `java.awt`. Referenced from `com.sun.jna.Native.AWT`.">
         <location
-            file="$GRADLE_USER_HOME/caches/transforms-3/61f2ace0377fec944d0761aa7634924b/transformed/jna-5.13.0/jars/classes.jar"/>
+            file="$GRADLE_USER_HOME/caches/transforms-3/a5c0a6fe0233ae59f2dafd8082c122a9/transformed/jna-5.13.0/jars/classes.jar"/>
```

Comment:
Here's what I meant @MiSikora in our [discussion](https://github.com/Automattic/pocket-casts-android/pull/2283/files#r1624254613).

Reply:
I see. It might be worth in that case to add `InvalidPackage` as an ignored issue to `lint.xml` file. What do you think?

Reply:
I think it's a good idea! I'll do this in scope of #2301 . The other baseline record that uses `$GRADLE_USER_HOME`, `ObsoleteLintCustomCheck` is resolved there (fixed).